ports/randungeon/port.json

```
@@ -0,0 +1,26 @@
+{
+    "version": 2,
+    "name": "randungeon.zip",
+    "items": [
+        "Randungeon.sh",
+        "randungeon"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Randungeon",
+        "porter": [
+            "TheLaggyPyro"
+        ],
+        "desc": "a simple roguelike make for a gamejam"
+        "inst": " locate and move pck file to gamedata.",
+        "genres": [
+            "adventure"
+        ],
+        "image": {},
+        "rtr": true,
```

Since the game is not "Ready to Run" as it requires downloading the game from gamejam, you'll want to set this to false:
```suggestion
        "rtr": false,
```
Likewise, there is no gamedata folder present here to place the game file into.  This isn't a problem for us testers, but may confuse the casual user.  Make sure to have the gamedata folder created and you may need to place an empty file inside the gamedata folder called .gitkeep so that git doesn't remove the directory.